client/thin-replica-client/src/thin_replica_client.cpp

```diff
@@ -501,6 +501,10 @@ void ThinReplicaClient::receiveUpdates() {
     LOG_WARN(logger_, "Need to stop receiving updates");
     return;
   }
+  // start time for metrics updates_received_per_second and updates_processed_per_second
+  std::chrono::steady_clock::time_point start_time = std::chrono::steady_clock::now();
```

Comment:
Suggestion: Rename to `start_per_second` because we already have a `start` time variable for the while loop.

Reply:
I have removed `start_per_second`, as it is no longer needed, since I have changed `updates_processed_per_second` -> `num_updates_processed` i.e., a counter.